### PR TITLE
Dev rc/v3 bug fix

### PR DESCRIPTION
**Pull request by remocristoforetti**
Minor bug fixing in:

- `matRad_writeMHD`
- `matRad_StfGenratorBase`

### CODE REVIEW

File: matRad/IO/matRad_writeMHD.m
Comment (wahln) [inline matRad/IO/matRad_writeMHD.m:1]: Are you sure that this is correct? Have you checked it in another Platform (e.g. Slicer / MITK)?

**Review by wahln — COMMENTED** [review]
Thanks for looking into it. Not entirely sure about the axis permutation fix, as it also needs to be seen in the bigger context of the writeCube function. Please make sure that this is consistent with what would be imported in another platform (e.g. 3D Slicer or MITK) and that the reimport in matRad works out (maybe the tests don't cover this situation yet).

File: matRad/IO/matRad_writeMHD.m
Comment (wahln) [reply matRad/IO/matRad_writeMHD.m:1]: If this fix is correct also some of the other writers seem to be wrong (vtk, mha)

Comment (remocristoforetti) [reply matRad/IO/matRad_writeMHD.m:1]: I'll have another look at this. It was giving me issues with phantoms that are not symmetric in x and y in indexing the dimensions correctly. I'll double check for possible further inconsistency downstream in my code causing the problem. Maybe don't merge until I further check this.

Comment (wahln) [reply matRad/IO/matRad_writeMHD.m:1]: yeah, just try to load an exported mha file that has inconsistent x/y in Slicer or MITK

**Review by wahln — APPROVED** [review]

**Pull request merged**